pkg/asset-manager-utils/contracts/IAssetManager.sol

```diff
@@ -73,8 +78,14 @@ interface IAssetManager {
 
     /**
      * @notice Rebalances funds between the pool and the asset manager to maintain target investment percentage.
-     * If the pool is below its critical threshold for the amount invested then calling this will send a small reward
-     * to the sender
+     * @param poolId - the poolId of the pool to be rebalanced
      */
     function rebalance(bytes32 poolId) external;
```

Comment:
I'd consider dropping this: it is equivalent to `rebalance(pId, true)`, and having overloads is a pain in client code.